### PR TITLE
Update plotting for ska3 and add light cron task

### PR DESCRIPTION
**Pull request by jeanconn**
Update plotting for ska3 and add light cron task

### CODE REVIEW

**Comment by jeanconn** [comment]
I cheated and just set the updated plots to write to the live page.  

http://cxc.harvard.edu/mta/ASPECT/obc_rate_noise/trending/

The last version of the old plots are in

http://cxc.harvard.edu/mta/ASPECT/obc_rate_noise/trending/backup

for a comparison.

File: task_schedule.cfg [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+data_dir     $ENV{SKA}/data//obc_rate_noise_plots       # Data file directory
+log_dir      $ENV{SKA}/data/obc_rate_noise_plots/logs  # Log file directory
+bin_dir      $ENV{SKA}/share/obc_rate_noise_plots      # Bin dir (optional, see task def'n)
+master_log   rate_noise.log             # Composite master log (created in log_dir)
+
+# Email addresses that receive an alert if there was a severe error in
+# running jobs (i.e. couldn't start jobs or couldn't open log file).
+# Processing errors *within* the jobs are caught with watch_cron_logs
+
+alert      jeanconn@head.cfa.harvard.edu
```
Comment (jeanconn) [inline task_schedule.cfg:30]: Will swap this over to aca when installed in ska3/flight .

**Comment by taldcroft** [comment]
:tada:  I'm not doing detailed code review but the output plots look basically fine.

What machine is this running on?  IIRC it takes a fair bit of memory.

**Review by taldcroft — APPROVED** [review]

**Comment by jeanconn** [comment]
I've got the dev job set to run on kadi early Wednesday mornings for now and will also update the twiki job page when promoted.  And yes, the jobs sucks up basically all the memory on fido to do the "longterm" plots but kadi still seems OK.

**Comment by taldcroft** [comment]
Perfect.

**Comment by jeanconn** [comment]
I note that there were some differences between the kadi dwells and the obs_kalman table entries that were used previously.  I wasn't rigorous in this review and I just noticed a *missing* entry in the obs_kalman Sybase table.  

I think at this point kadi.dwells are the reference so I didn't dwell on this much.  Pun intended.

**Comment by taldcroft** [comment]
Even better, I had not noticed the first time through all the code deletion and getting rid of a sybase dependency.  💯 👍

**Pull request merged**